src/guide/hotspot-development.md

```diff
@@ -1,5 +1,10 @@
 # HotSpot Development
 
+See [Working With Pull Requests](#working-with-pull-requests) for generic guidance and requirements around pushing changes. For the HotSpot codebase there are a few additional requirements:
+
+* Your change must have been approved by two Committers out of which at least one is also a Reviewer
+* Your change must have passed through HotSpot tier 1 testing with zero failures (See tier1 definition in `test/hotspot/jtreg/TEST.groups`.)
+
```

Comment:
I thought the 24 hour rule was part of the HotSpot tailoring (present in both the internal and external HotSpot review guidelines), rather than a JDK-wide requirement.  But it's in the generic requirements in the Guide.  Am I misremembering, or is that a change I didn't notice before?

Reply:
And I don't see the 24 hour rule being strictly adhered to elsewhere.  For example, I see the occasional quick turnaround in core-libs changes.

Reply:
I think the 24 hour rule was hotspot specific. Other areas do very quick turn-arounds.

Reply:
I can't say that I have studied what happens in reality in other areas, but the 24 hour text was added as a general rule in "Working with pull requests" in PR #79 which is one of the more reviewed PRs in the Guide. Several developers from all areas of the JDK was involved and no one had any issues with the rule. In fact, the only comment about that particular guidance was from a person active in the libraries area and he suggested to add the sentence about holidays etc to say that sometimes even 24 hours isn't enough. So it seems to me that this is an accepted rule in general. Wether people actually follow the rule may be a different story.

Reply:
FWIW, I think the 24 hour "rule" is a good one for all component teams, except I think it should be more like a best practice, rather than a hard rule. Exceptions would be for trivial fixes and fixes for build errors or tier 1 or tier 2 failures where a quicker turnaround is necessary.

Reply:
I think the current version of the new text is fine given that the requirements are preceded by the phrase "For the HotSpot codebase...".